installation-tests/initialize_test.sh

```diff
@@ -3,6 +3,8 @@
 # break script execution if some command returns non-zero exit code
 set -e
 
+TEST_FRAMEWORK_RUN_ROOT="/tmp/playwright-installation-tests"
```

Comment:
Should we use something like os.tmpdir()?

Reply:
it'll make it hard to debug test failures, so I'd rather not.